tests/integration/client.py

```diff
@@ -151,6 +154,13 @@ def configure():
         'hil.ext.network_allocators.vlan_pool': {
             'vlans': '1001-1040',
         },
+        'devel': {
+            # Disable dry_run, so we can talk to obmd. Note: We register
+            # several "real" switches in this module, but never actually
```

Comment:
I have changed a lot of this stuff in my PR #1014. If that gets merged first, you might have to change a lot of things in this module. What do you think we should do first?

Reply:
I will deal with the conflicts. I don't think they'll be too horrible.